contentcuration/contentcuration/frontend/channelEdit/views/files/FileUpload.vue

```diff
@@ -140,17 +140,17 @@
       this.selectFirstFile();
     },
     methods: {
-      ...mapActions('file', ['createFile', 'deleteFile']),
+      ...mapActions('file', ['updateFile', 'deleteFile']),
       selectFirstFile() {
         let firstFile = sortBy(this.files, f => f.preset.order)[0];
         this.selected = firstFile && firstFile.id;
       },
-      handleUploading(fileUpload) {
-        this.createFile({
-          contentnode: this.nodeId,
+      handleUploadComplete(fileUpload) {
+        this.updateFile({
           ...fileUpload,
-        }).then(id => {
-          this.selected = id;
+          contentnode: this.nodeId,
```

Comment:
Just to clarify, is this contentnode.pk or contentnode.node_id?

Reply:
This is the pk - I guess for consistency we should be saying `nodeNodeId` in places where we mean `node_id`.